definitions/modules/electrical.yaml

```diff
@@ -0,0 +1,117 @@
+# This file only supports aircraft with a max of two engines. You can specify more in the individual profile.
+
+shared:
+  - # ASOBO_ELECTRICAL_Switch_APU_Generator_Template
+    type: ToggleSwitch
+    var_name: A:APU GENERATOR SWITCH:1
+    var_units: Bool
+    event_name: APU_GENERATOR_SWITCH_TOGGLE
+    event_param: 1
+  -
+    type: ToggleSwitch
+    var_name: A:APU GENERATOR SWITCH:2
+    var_units: Bool
+    event_name: APU_GENERATOR_SWITCH_TOGGLE
+    event_param: 2
+
+  - # ASOBO_ELECTRICAL_Switch_Battery_Master_Template
+    type: ToggleSwitch
+    var_name: A:ELECTRICAL MASTER BATTERY:1
+    var_units: Bool
+    event_name: TOGGLE_MASTER_BATTERY
+    event_param: 1
+  -
+    type: ToggleSwitch
+    var_name: A:ELECTRICAL MASTER BATTERY:1
+    var_units: Bool
+    event_name: TOGGLE_MASTER_BATTERY_ALTERNATOR
+    event_param: 1
+  -
+    type: ToggleSwitch
+    var_name: A:ELECTRICAL MASTER BATTERY:2
+    var_units: Bool
+    event_name: TOGGLE_MASTER_BATTERY
+    event_param: 2
+  -
+    type: ToggleSwitch
+    var_name: A:ELECTRICAL MASTER BATTERY:2
+    var_units: Bool
+    event_name: TOGGLE_MASTER_BATTERY_ALTERNATOR
+    event_param: 2
+
+  - # ASOBO_ELECTRICAL_Switch_Battery_STBY_Template
+    type: ToggleSwitch
+    var_name: A:ELECTRICAL MASTER BATTERY:3
+    var_units: Bool
+    event_name: TOGGLE_MASTER_BATTERY
+    event_param: 3
+
+  - # ASOBO_ELECTRICAL_Switch_Avionics_Master_Template
+    type: ToggleSwitch
+    var_name: A:AVIONICS MASTER SWITCH
+    var_units: Bool
+    event_name: TOGGLE_AVIONICS_MASTER
+
+  - # ASOBO_ELECTRICAL_Switch_Alternator_Template
+    type: ToggleSwitch
+    var_name: A:GENERAL ENG MASTER ALTERNATOR:1
+    var_units: Bool
+    event_name: TOGGLE_ALTERNATOR1
+  -
+    type: ToggleSwitch
+    var_name: A:GENERAL ENG MASTER ALTERNATOR:1
+    var_units: Bool
+    event_name: TOGGLE_MASTER_ALTERNATOR
+  -
+    type: ToggleSwitch
+    var_name: A:GENERAL ENG MASTER ALTERNATOR:2
+    var_units: Bool
+    event_name: TOGGLE_ALTERNATOR2
+
+  - # ASOBO_ELECTRICAL_Switch_ExternalPower_Template
+    type: ToggleSwitch
+    var_name: A:EXTERNAL POWER AVAILABLE:1
+    var_units: Bool
+    event_name: TOGGLE_EXTERNAL_POWER
+    event_param: 1
```

Comment:
external power available doesn't mean it's on so it shouldn't turn on the external power...
```suggestion
```

Reply:
The switch lights up when it detects external power is connected, hence the name "available" and able to be toggled on. The 787 EXTERNAL POWER ON, to toggle on.

Reply:
yes *able to be toggled on* but not toggled on yet. this definitions implies that `TOGGLE_EXTERNAL_POWER` should be sent when it is only *available* and not *on* yet. delete this definition and leave the definition below.